src/mtcp/mtcp_util.h

```diff
@@ -35,11 +35,11 @@
     mtcp_abort();                                       \
   }
 
-#ifdef DEBUG
+#ifdef LOGGING
```

Comment:
Is the `LOGGING` macro available in mtcp? Do we need to change `Makefile.in` in the mtcp directory? Could you also see if we need to fix Makefile.debug in the mtcp directory?